add-ons/istio/README.adoc

```diff
@@ -0,0 +1,62 @@
+:linkattrs:
+
+= Istio Add-on
+
+An addon that will deploy https://Istio.io/[Istio] service mesh on to OpenShift provided by Minishift.
+
+Verify that you have installed the add-on by following the https://github.com/minishift/minishift-addons[Minishift add-ons]
+
+== Minishift profile setup
+
+As https://Istio.io/[Istio] setup spins up multiple pods as part of the setup,  hence its ideal to have a 
+setup a minishift profile with the following configuration to avoid any resource crunch.
+
+[code,sh]
+----
+$ minishift profile set servicemesh <1>
+$ minishift config set memory 8GB <2>
+$ minishift config set cpus 3 <3>
+$ minishift config set openshift-version v3.7.0 <4>
+$ minishift addon enable admin-user <5>
+$ minishift addon enable anyuid <6>
+$ minishift start <7>
+----
+
+<1> Create a new Minishift profile in the command its called servicemesh
+<2> Set a memory of 8GB, this is good amount of memory for istio and apps that you might deploy later
```

Comment:
A "good amount of memory" feels a bit like a value judgement, even though it technically isn't one.

`<2> Use 8GB of memory for Istio and apps that may be deployed later`